### PR TITLE
Add newtype Cusno in KSF.User.Cusno

### PR DESCRIPTION
**Pull request by kaol**

### CODE REVIEW

File: packages/components/src/Navbar/Component.purs [hunk 1]
```diff
@@ -179,7 +180,7 @@ customerService isPersonating activeUser = do
       userLink user =
         Router.link
           { to: { pathname: "/", state: {} }
-          , children: [ DOM.text $ user.cusno <> " - " <>
+          , children: [ DOM.text $ (show $ (\(Cusno c) -> c) $ user.cusno) <> " - " <>
```
Comment (ae-mo) [inline packages/components/src/Navbar/Component.purs:183]: I think `show $ (\(Cusno c) -> c` could become a `toString` or `fromCusno` function to put in the `KSF.User.Cusno` module.

File: apps/mitt-konto/src/MittKonto/Main/Views/UserView/Subscription/Elements.purs [hunk 2]
```diff
@@ -275,11 +275,11 @@ subscriptionUpdates self@{ props: props@{ subscription: sub@{ subsno, package }
                           self.setState _
                             { wrapperProgress = AsyncWrapper.Success $ Just "Tillfällig adressändring har tagits bort",
                             pendingAddressChanges = toMaybe newSubscription.pendingAddressChanges }
-                          Tracking.deleteTempAddressChange (show props.subscription.cusno) (show props.subscription.subsno) startDate' endDate' "success"
+                          Tracking.deleteTempAddressChange props.subscription.cusno (show props.subscription.subsno) startDate' endDate' "success"
```
Comment (ae-mo) [inline apps/mitt-konto/src/MittKonto/Main/Views/UserView/Subscription/Elements.purs:278]: I think we should add the `Subsno` type as well, but we can do it in a different PR.

**Review by ae-mo — CHANGES_REQUESTED** [review]

File: apps/mitt-konto/src/MittKonto/Main/Views/UserView/Subscription/Elements.purs (hunk 2 above)
Comment (kaol) [reply apps/mitt-konto/src/MittKonto/Main/Views/UserView/Subscription/Elements.purs:278]: I would leave it for a later one. I did this for `Cusno`s now since I needed to do something to get to compare `Cusno`s in subscriptions and in user data.

File: packages/components/src/Navbar/Component.purs (hunk 1 above)
Comment (kaol) [reply packages/components/src/Navbar/Component.purs]: I added `toString`. Using plain `show` in code is always suspicious anyway.

**Review by ae-mo — APPROVED** [review]
